ProfessorPage.html

```diff
@@ -20,23 +21,29 @@
 	
 <body>
 	<!-- navbar -->
-	<nav class="navbar navbar-inverse">
-	<div class="container-fluid">
-		<div class="navbar-header">
-		<a class="navbar-brand" href="index.html">
-			<img src=hat.png style="background-color:white;" width="30" height="30" class="d-inline-block align-top" alt="">
+	<nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+		<a class="navbar-brand" href="#">
+			<img src="hat.png" alt="Logo" style="width:40px;">
 		</a>
-		</div>
-		<ul class="nav navbar-nav">
-			<li class="active"><a href="#">Home</a></li>
-			<li><a href="#">Page 1</a></li>
-			<li><a href="#">Page 2</a></li>
+		<ul class="navbar-nav mr-auto">
+			<li class="nav-item">
+				<a class="nav-link" href="#">Home</a>
+			</li>
+			<li class="nav-item">
+				<a class="nav-link" href="#">Page 1</a>
+			</li>
+			<li class="nav-item">
+				<a class="nav-link" href="#">Page 2</a>
+			</li>
 		</ul>
-		<ul class="nav navbar-nav navbar-right">
-			<li><a id ="userAvatar" href="#"><span class="glyphicon glyphicon-user"></span> User </a></li>
-			<li><a href="#" onclick="signOut()" ><span id="logout" class="glyphicon glyphicon-log-out"></span> Logout</a></li>
+		<ul class="navbar-nav ml-auto">
+			<li class="nav-item">
+				<a class="nav-link" id="userNavbarLink" href="#"><span class="fa fa-user"></span> User</a>
```

Comment:
I have JS methods that are dependent on the name of this element. See line 21 of studentPage.js. Either you have to keep these as "userAvatar" or change all of my JS files. The JS gets the user's name and replaces the text of this element with the username.